docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,654 @@
+{{< /alert >}}
+
+Now that you have added a Recipe, run this command to display the state of the Radius application:
+
+```bash
+rad app graph -a demo -g default-demo
+```
+
+The output should look like this:
+
+```txt
+> rad app graph -a demo -g default-demo
+Displaying application: demo
+
+Name: webapp (Applications.Core/containers)
+Connections: (none)
+Resources:
+  webapp (kubernetes: apps/Deployment)
+
+Name: db (Applications.Datastores/redisCaches)
+Connections: (none)
+Resources:
+  redis-r5tcrra3d7uh6 (kubernetes: apps/Deployment)
+  redis-r5tcrra3d7uh6 (kubernetes: core/Service)
+
+
+```
+
+`rad app graph` shows the **Application Graph** of the application. This includes:
+
+- Entries for each major resource: `webapp` is an `Applications.Core/containers` and `db` is an `Applications.Datastores/redisCaches`.
+- Connections between resources: (none yet, you will add this next).
+- Resources that were created: see the Kubernetes `Deployment` listed for `webapp` and the Kubernetes `Deployment` and `Service` listed for `db`.
+
+The Redis Cache created by the recipe is visible as part of the application. You can also see the `Resources` created in Kubernetes that make up the Redis Cache. In a previous step you saw these listed by `kubectl`. Since Radius deployed the Recipe its know that these resources *logically* are part of the Redis Cache in the application.
+
+You can also see the contents of `redis-secret` as created by Radius. Run the following command:
+
+```bash
+kubectl get secret -n demo redis-secret -o yaml
+```
+
+The output should look like the following:
+
+```txt
+>kubectl get secret -n demo redis-secret -o yaml
+apiVersion: v1
+data:
+  connectionString: cmVkaXMtcjV0Y3JyYTNkN3VoNi5kZW1vLnN2Yy5jbHVzdGVyLmxvY2FsOjYzNzksYWJvcnRDb25uZWN0PUZhbHNl
+  host: cmVkaXMtcjV0Y3JyYTNkN3VoNi5kZW1vLnN2Yy5jbHVzdGVyLmxvY2Fs
+  password: ""
+  port: NjM3OQ==
+  tls: ZmFsc2U=
+  url: cmVkaXM6Ly9yZWRpcy1yNXRjcnJhM2Q3dWg2LmRlbW8uc3ZjLmNsdXN0ZXIubG9jYWw6NjM3OS8wPw==
+  username: ""
+kind: Secret
+metadata:
+  creationTimestamp: "2023-09-13T01:49:36Z"
+  name: redis-secret
+  namespace: demo
+  ownerReferences:
+  - apiVersion: radapp.io/v1alpha3
+    blockOwnerDeletion: true
+    controller: true
+    kind: Recipe
+    name: db
+    uid: d40567a1-cd52-4984-8321-6cb8bea5f798
+  resourceVersion: "3672"
+  uid: b1613fb0-09e6-4f76-8685-02f458e173b9
+type: Opaque
+```
+
+The actual values like `connectionString` are Base64 encoded in this display. The `url` value in this secret is being used by the container to connect to the Redis Cache. For each type of Recipe, Radius stores the most-commonly used connection information for the convenience of application developers.
+
+
+## 6. Add Connection
+
+At this point we have added Radius to our existing container, and used a Recipe to create a Redis Cache. In this step you will use Radius Connections to inject settings into the container instead of explicitly managing a secret.
```

Comment:
```suggestion
At this point you have added Radius to your existing container and used a Recipe to create a Redis Cache. In this step, you will use Radius Connections to inject settings into the container instead of explicitly managing a secret.
```